org.eclipse.dartboard/src/org/eclipse/dartboard/preference/DartSDKLocationFieldEditor.java

```diff
@@ -75,9 +78,13 @@ private boolean isValidDartSDK(String location) {
 		} catch (InvalidPathException e) {
 			return false;
 		}
+
+		// See https://github.com/eclipse/dartboard/issues/103
+		List<String> blacklist = Lists.newArrayList("/bin/dart", "/usr/bin/dart");
```

Comment:
Why do we need to add blacklist? Instead of that, we can just check the expected path right?.
I mean,  {input_path}/bin/dart(.exe) file exist?

Reply:
No, because /bin/dart exists on the file system... It's a symlink for /usr/lib/dart/bin/dart. Same with /usr/bin/dart.

Also, the blacklist is more a convenience as there might be more paths (on other OSes), that we have to cover, and now we only need to add them to this list.